orderapi/src/main/java/fi/hel/verkkokauppa/order/model/refund/RefundItem.java

```diff
@@ -12,6 +13,7 @@
 @Document(indexName = "refunditems")
 @Getter
 @Setter
+@NoArgsConstructor
```

Comment:
FYI: @Getter ,@Setter, @NoArgsConstructor can be replaced with one annotation @Data